Refresh.GameServer/Services/GuidCheckerService.cs

```diff
@@ -43,6 +47,9 @@ private static void ReadStream(Stream stream, HashSet<long> set)
     /// <exception cref="ArgumentOutOfRangeException"></exception>
     public bool IsTextureGuid(TokenGame game, long guid)
     {
+        // Check if all guids are enabled
+        if (this._config.EnableUnknownGuids) return true;
```

Comment:
More bikeshedding, but i'd prefer if it was 
```suggestion
        // If specified in the config, allow all GUIDs
        if (this._config.EnableUnknownGuids) return true;
```